include-dependencies.js

```diff
@@ -53,10 +55,31 @@ module.exports = class IncludeDependencies {
     service.package.exclude = union(service.package.exclude, ['node_modules/**']);
 
     if (runtime.match(/nodejs*/)) {
+      this.processIncludes(functionObject);
       this.processNodeFunction(functionObject);
     }
   }
 
+  processIncludes(functionObject) {
+    const service = this.serverless.service;
+    if (!service.package || !service.package.include || !service.package.include.length) {
+      return;
+    }
+    service.package.include.forEach(includeGlob => {
+      glob.sync(path.join(this.serverless.config.servicePath, includeGlob)).forEach(filePath => {
```

Comment:
Should we filter includeGlobs that match `node_modules` ? I'm not sure about the behavior

Reply:
(I wonder if this would improve the perf)

Reply:
For my particular use case, I'm not including globs from `node_modules`.